modules/virt-configuring-certificate-rotation.adoc

```diff
@@ -0,0 +1,42 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-configuring-certificate-rotation.adoc
+
+[id="virt-configuring-certificate-rotation_{context}"]
+
```

Comment:
You are missing the title, which is why this isn't showing up in the ToC. Titles go directly under the ID like this:

```
[id="virt-configuring-certificate-rotation_{context}"]
= Configuring certificate rotation
```